src/firmware/stm32f405.ld

```diff
@@ -57,6 +57,13 @@ SECTIONS {
 	linker_heap_size = 128K - (. - 0x20000000);
 	.heap : ALIGN(8) {
 		. = linker_heap_size;
+        __heap_start__ = .;
```

Comment:
is this autogenerated by the tool?

Reply:
Nope, we wrote our own for the old firmware (as best I can tell). For the new firmware it is generated. I fixed an issue with the old firmware by comparing some bits of it to the new linker script.